dvc/stage.py

```diff
@@ -128,7 +129,8 @@ class Stage(object):
     PARAM_META = "meta"
 
     SCHEMA = {
-        Optional(PARAM_MD5): Or(str, None),
+        Optional(hash.CHECKSUM_MD5): Or(str, None),
```

Comment:
As @shcheklein noted before, let's define `CHECKSUM_SCHEMA` for example in `dvc/utils/checksum.py`(alongside file_checksum(), see my comment about renaming dvc/utils/hash/__init__.py down below) and use it in Stage SCHEMA and Output SCHEMA